### PR TITLE
Add Sportico to config

### PR DESCRIPTION
**Pull request by sarahsmo**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ sites:
       - pmc-deadline-2019.wpcom.test
       - pmc-footwearnews-2018.wpcom.test
       - pmc-hollywoodlife-2017.wpcom.test
+      - pmc-sportico-2020.wpcom.test
```
Comment (ethitter) [inline config.yml:13]: Sportico shouldn't be included here because it isn't a VIP Classic site.

File: config.yml [hunk 2]
```diff
@@ -51,6 +52,30 @@ sites:
...
+      delete_default_plugins: true
+      install_test_content: true
+      wpconfig_constants:
+        AUTOMATIC_UPDATER_DISABLED: true
+        DISALLOW_FILE_EDIT: true
+        DISALLOW_FILE_MODS: true
+        WP_DEBUG: false
+        WP_DEBUG_LOG: false
+        WP_DEBUG_DISPLAY: false
+        WP_DISABLE_FATAL_ERROR_HANDLER: false
```
Comment (ethitter) [inline config.yml:71]: We'll need these constants as well:

```
PMC_IS_VIP_GO_SITE: true
VIP_GO_APP_ENVIRONMENT: true
VIP_GO_ENV: true
```

`PMC_IS_VIP_GO_SITE` is set based on `VIP_GO_ENV`, but there's no harm in being explicit in VVV.

**Review by ethitter — CHANGES_REQUESTED** [review]

File: config.yml (hunk 1 above)
Comment (sarahsmo) [reply config.yml]: removed!

File: config.yml (hunk 2 above)
Comment (sarahsmo) [reply config.yml:70]: added! Thanks!

**Review by ethitter — APPROVED** [review]
:shipit:

**Pull request merged**